Ansible/inventory

```diff
@@ -0,0 +1,10 @@
+all:
+  hosts:
+    vm2:
+      ansible_host: 192.168.56.12
+      ansible_user: vagrant
+      ansible_ssh_pass: vagrant
+      ansible_ssh_private_key_file: ~/.ssh/id_rsa
+     # admin_user: matusik
+     # admin_ssh_pass: matusik
+     # admin_ssh_private_key_file: ~/.ssh/id_rsa
```

Comment:
Remove all lines that are commented and unnecessary, and do not add any value to the code.
Check the whole PR